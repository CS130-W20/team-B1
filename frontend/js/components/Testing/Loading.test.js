import React from 'react';
import renderer from 'react-test-renderer';

import Loading from '../Loading';

describe ('Testing Loading Page', () => {
    it('correctly renders', () => {
        const wrapper = renderer.create(<Loading />).toJSON();
        expect(wrapper).toMatchSnapshot();
    });
});
