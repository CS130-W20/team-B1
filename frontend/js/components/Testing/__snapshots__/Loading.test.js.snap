// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Loading Page correctly renders 1`] = `
Array [
  <div
    className="spinner-grow text-primary"
    style={
      Object {
        "animationDuration": "2.1s",
      }
    }
  />,
  <div
    className="spinner-grow text-secondary"
    style={
      Object {
        "animationDelay": "0.3s",
        "animationDuration": "2.1s",
      }
    }
  />,
  <div
    className="spinner-grow text-success"
    style={
      Object {
        "animationDelay": "0.6s",
        "animationDuration": "2.1s",
      }
    }
  />,
  <div
    className="spinner-grow text-danger"
    style={
      Object {
        "animationDelay": "0.9s",
        "animationDuration": "2.1s",
      }
    }
  />,
  <div
    className="spinner-grow text-warning"
    style={
      Object {
        "animationDelay": "1.2s",
        "animationDuration": "2.1s",
      }
    }
  />,
  <div
    className="spinner-grow text-info"
    style={
      Object {
        "animationDelay": "1.5s",
        "animationDuration": "2.1s",
      }
    }
  />,
]
`;
